backend/src/routes/squad.js

```diff
@@ -1,7 +1,8 @@
 const express = require("express");
 
 const squadController = require('../controllers/squad');
-const { isAuth, attachCurrentUser } = require('../middleware');
+const isAuth = require('../middleware/isAuth').isAuth;
+const attachCurrentUser = require('../middleware/attachCurrentUser');
```

Comment:
No love for ES6 destructuring? I'm happy with it either way, as long as we're consistent. If we go this way, we can remove all the `index.js`'s from a few directories.

Reply:
I honestly cannot figure it out. If you want to make a small PR to fix it then go for it haha. But I've spent like 20 minutes trying to destructure things and export as default but it's just not working.